### ISSUE TITLE
[Loader] Accessibility

### ISSUE THREAD

**Issue by MartinWeb** [AxaFrance/react-toolkit#1053]
### Issue and Steps to Reproduce

When using the loader, no information is read by a screen reader. The loader should announce that something is currently in action so that the user doesn't find himself in a situation where he cannot use the site for a certain amount of time but not knowing it.

#### Versions

2.0.0
